R/confirmatoryfactoranalysis.R

```diff
@@ -985,21 +985,23 @@ ConfirmatoryFactorAnalysis <- function(jaspResults, dataset, options, ...) {
       DoNotPlot      = TRUE,
       ask            = FALSE,
       layout         = "tree",
+      rotation       = ifelse(options$plotrotate, 2, 1),
       intercepts     = options$plotmeans,
       whatLabels     = if (!options$plotpars) "name" else if (options$plotstd) "std" else "par",
-      mar            = ifelse(rep(is.null(options$secondOrder), 4), c(12, 3, 12, 3), c(6, 3, 6, 3)),
       edge.color     = "black",
       color          = list(lat = "#EAEAEA", man = "#EAEAEA", int = "#FFFFFF"),
       border.width   = 1.5,
       edge.label.cex = 0.9,
```

Comment:
We could think about enabling the user change this argument (`edge.label.cex`) as this probably relates to readability the most...